### PR TITLE
[SEA] Temenos

### PR DESCRIPTION
**Pull request by nesstea**
Yeaaahhhhhhh!

### CODE REVIEW

**Comment by TeoTwawki** [comment]
Actually sanity and proper indention? In the limbus files?
 WORLD IS ENDING!  *-holds up a sign on a street corner-*

File: scripts/zones/Port_Jeuno/npcs/Sagheera.lua [hunk 1]
```diff
@@ -11,57 +11,57 @@ require("scripts/zones/Port_Jeuno/TextIDs");
...
+            if (trade:hasItemQty(Relic_Armor_Plus_one[nb][2],1) and  trade:hasItemQty(Relic_Armor_Plus_one[nb][3],1) and  trade:hasItemQty(Relic_Armor_Plus_one[nb][4],1)  and  trade:hasItemQty(Relic_Armor_Plus_one[nb][5],1) and Relic_Armor_Plus_one[nb][6] <= StoreAncientBeastcoins) then 
+                AvaibleCombinaisonDectect = Relic_Armor_Plus_one[nb-1];
+                cost = Relic_Armor_Plus_one[nb][6];
+                printf("detect trade  avaible relic combinaison: %u", Relic_Armor_Plus_one[nb][1]);   
+            end
+        end
+    elseif (CurrentAFupgrade == 0 and AvaibleCombinaisonDectect == 0) then -- Artfact Armor +1 ???
+        for nb = 2, table.getn(Artifact_Armor_Plus_one), 2 do  --looking for the Artifact armor
+            --trade base Artfact armor                                                                  --- trade Artfact armor -1                                                                       trade craft item
```
Comment (xdemolish) [inline scripts/zones/Port_Jeuno/npcs/Sagheera.lua]: corrects spelling with typo
win

Comment (teschnei) [inline scripts/zones/Port_Jeuno/npcs/Sagheera.lua]: Since you're here anyways, Sagheera needs to use getCurrency instead of her
own player var

**Comment by nesstea** [comment]
Ah yeah I'll do that.

**Pull request merged**